addons/mail/models/mail_activity.py

```diff
@@ -389,6 +392,35 @@ def _compute_display_name(self):
             name = record.summary or record.activity_type_id.display_name
             record.display_name = name
 
+    # ------------------------------------------------------
+    # Notifications
+    # ------------------------------------------------------
+
+    def _notify_owner_store(self, store, notification_type="mail.record/update_activity"):
+        self.ensure_one()
+        if not self.res_model_id:
+            return
+        owner_model = self.env[self.res_model].browse(self.res_id)
+        data = {
+            "res_id": self.res_id,
+            "res_model": self.res_model,
+            "data": store.get_result(),
+        }
+        owner_model._bus_send(notification_type, data)
+
+    def notify_owner_activity(self, activities=None):
```

Comment:
I don't think we want to expose those methods to the public API. Also, I'm not sure about splitting it into 3 methods. Why not `_notify_owner_store(unlink=True)` (not sure about the best parameter but you get the idea :))